.github/workflows/tests.yml

```diff
@@ -174,30 +174,27 @@ jobs:
 
     - name: Build Docker image
       if: matrix.os == 'ubuntu-latest'
-      run: |
-        docker pull homebrew/brew
-        docker-compose -f Dockerfile.yml build sut
+      run: docker build -t brew .
 
     - name: Run brew test-bot
       run: |
         if [ "$RUNNER_OS" = "Linux" ]; then
-          docker-compose -f Dockerfile.yml run --rm -v $GITHUB_WORKSPACE:/tmp/test-bot sut
-          docker tag homebrew_sut brew
+          docker run --rm -v $GITHUB_WORKSPACE:/tmp/test-bot brew brew test-bot
         else
           brew test-bot
         fi
 
-    - name: Deploy the latest Docker image to GitHub
-      if: matrix.os == 'ubuntu-latest' && github.ref == 'refs/heads/master'
-      run: |
-        docker login docker.pkg.github.com -u BrewTestBot -p ${{secrets.GITHUB_TOKEN}}
-        docker tag brew docker.pkg.github.com/homebrew/brew/brew
-        docker push docker.pkg.github.com/homebrew/brew/brew
-
-    - name: Deploy the tagged Docker image to GitHub
-      if: matrix.os == 'ubuntu-latest' && startsWith(github.ref, 'refs/tags/')
+    - name: Deploy the Docker image to GitHub and Docker Hub
```

Comment:
How long does this take?

Reply:
33 seconds. See https://github.com/Homebrew/brew/runs/643203256
Deployment doesn't happen in the PR check though, only for `master` and tags.

Reply:
Cool, sounds good, thanks 👍